packages/rspack/src/builtin-plugin/HtmlRspackPlugin.ts

```diff
@@ -378,6 +378,7 @@ const compilationOptionsMap: WeakMap<Compilation, HtmlRspackPluginOptions> =
 	new WeakMap();
 
 const HtmlRspackPlugin = HtmlRspackPluginImpl as typeof HtmlRspackPluginImpl & {
+	getHooks: (compilation: Compilation) => HtmlRspackPluginHooks;
```

Comment:
Should we mark `getHooks` as deprecated method? The same as https://github.com/jantimon/html-webpack-plugin/blob/41f4a7b952e80eea4878d96638b4021f4a762750/index.js#L1593